style.css

```diff
@@ -0,0 +1,918 @@
+  margin-bottom: 11vh;
+  margin-left: 6vw;
+}
+
+.paragraph-two h3 {
+  font-family: lato, sans-serif;
+  font-size: 5vw;
+  color: rgb(14, 13, 13, 0.85);
+  margin-top: 1px;
+}
+
+.px {
+  padding-left: 30vw;
+}
+
+.paragraph-two p {
+  font-family: lato, sans-serif;
+  position: absolute;
+  top: 2.5vh;
+  font-size: 3.5vw;
+  color: rgb(21, 35, 35);
+}
+
+.program {
+  background: url(./img/background-program.jpg) no-repeat center center fixed;
+  background-size: cover;
+}
+
+.title-program {
+  display: flex;
+  justify-content: center;
+}
+
+h5 {
+  display: flex;
+  font-family: lato, sans-serif;
+  justify-content: center;
+  color: white;
+  font-size: 3vh;
+  position: relative;
+  margin-bottom: 2px;
+  padding-top: 2vh;
+}
+
+hr {
+  border-width: 2px;
+  color: red;
+  width: 8vw;
+}
+
+.one {
+  margin-left: 19vw;
+}
+
+h4 {
+  font-family: lato, sans-serif;
+  padding-top: 1vh;
+  padding-right: 9vw;
+  padding-left: 9vw;
+  font-size: 3vh;
+  color: rgb(36, 35, 35);
+  text-align: center;
+  margin-bottom: 1px;
+}
+
+.about-one {
+  margin-top: 1vh;
+}
+
+.about-two {
+  margin-top: 8vh;
+  border-bottom: rgba(221, 11, 11, 0.65) solid 1px;
+  background-color: white;
+}
+
+.about-img {
+  width: 85vw;
+  border: steelblue 1px solid;
+  border-radius: 5px;
+  height: 25vh;
+  margin-left: 7vw;
+  background-image: url(./img/about-img3.png);
+}
+
+.about-img-logo {
+  width: 53vw;
+  border: red 2px solid;
+  border-radius: 2px;
+  height: 25vh;
+  margin-left: 6vw;
+  margin-bottom: 10vh;
+  padding-left: 14vw;
+  padding-right: 14vw;
+}
+
+.about-four {
+  height: 25vh;
+  background: rgb(29, 28, 28);
+  margin-left: -5vw;
+  margin-right: -5vw;
+}
+
+.mobile-program {
+  display: grid;
+  grid-template-columns: 30% 30% 40%;
+  border: rgb(46, 40, 36) solid 1px;
+  margin-right: 5vw;
+  background-color: rgba(59, 62, 69, 0.5);
+  margin-bottom: 2vh;
+  border-radius: 5px;
+}
+
+.mobile-ul {
+  list-style-type: none;
+  padding-left: 20px;
+}
+
+.titlex {
+  margin-left: 1px;
+}
+
+h6 {
+  font-family: lato, sans-serif;
+  color: red;
+  font-size: 3.5vw;
+  display: flex;
+  margin-top: 5vh;
+  margin-left: 2vh;
+  margin-right: 2vh;
+}
+
+.paragraph-program {
+  margin-top: 27px;
+  font-family: lato, sans-serif;
+  color: antiquewhite;
+  font-size: 2.3vw;
+  margin-left: 10vw;
+}
+
+.paragraph-model-about {
+  display: flex;
+  margin-top: 1px;
+}
+
+.icon-program {
+  max-width: 100%;
+  max-height: 100%;
+  padding: 3vw;
+}
+
+.button-program {
+  display: flex;
+  justify-content: center;
+}
+
+.logo-paragraph {
+  font-family: lato, sans-serif;
+  font-size: 4vw;
+  text-align: center;
+  margin-left: 5vw;
+  margin-right: 5vw;
+  margin-bottom: 5vh;
+  margin-top: 5vh;
+  padding-left: 9vw;
+  padding-right: 9vw;
+  line-height: 4.5vh;
+}
+
+.title-models {
+  display: flex;
+  justify-content: center;
+  height: 10vh;
+}
+
+.img-models {
+  width: 22vw;
+  border-radius: 2px;
+  height: 15vh;
+  margin: 3vh;
+}
```

Comment:
- [ ] Images should protect their ratio in mobile and desktop resolutions. In your implementation, Images stretch in mobile and higher resolutions. Please be sure images behave as seen in the design template.

Mobile
![Screenshot 2021-07-03 at 01-31-48 Polanco Studio](https://user-images.githubusercontent.com/1642014/124336320-35499180-dba6-11eb-8702-d76f98833de4.png)
Desktop
![Screenshot 2021-07-03 at 01-32-04 Polanco Studio](https://user-images.githubusercontent.com/1642014/124336327-3a0e4580-dba6-11eb-975a-fe50b5cc6e69.png)